net-mgmt/zabbix-agent/src/opnsense/mvc/app/models/OPNsense/ZabbixAgent/Menu/Menu.xml

```diff
@@ -0,0 +1,12 @@
+<menu>
+    <Services>
+      <ZabbixAgent VisibleName="Zabbix Agent" cssClass="fa fa-certificate fa-fw">
+          <Settings order="10" url="/ui/zabbixagent/">
+              <MainSettings VisibleName="Main Settings" url="/ui/zabbixagent/#subtab_zabbixagent-settings-main"/>
```

Comment:
Level 1: Services
Level 2: Zabbix Agent
Level 3: Settings
Level 4: Main Settings

I don't think we ever had the "Level 4" in the menu?